contracts/helpers/LunarPrizeTiers.sol

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: -- BCOM --
+
+pragma solidity =0.8.21;
+
+contract PrizeTiers {
```

Comment:
looks good, but now we have 2 contracts PrizeTiers, not sure which one is inherited. 
- might be better to follow conversion that contract name always matches file name so `LunarPrizeTiers` and need import and inherit correct one.

Reply:
temporary resolved in: https://github.com/bitcoin-verse/verse-scratch-cards-contracts/commit/4f34ccee7c83b9350fcf280c44258bbb16142512

might find a better way to inherit a separate contract/wrapper and/or assign different tiers without modifying existing singletone version. (abillity to create variations with different tiers)